fhir-persistence-schema/src/test/java/com/ibm/fhir/schema/control/VersionHistoryServiceTest.java

```diff
@@ -64,4 +57,12 @@ public void addTestHistory(String objectSchema, String objectType, String object
     public static String makeKey(String objectSchema, String objectType, String objectName) {
         return objectSchema + ":" + objectType + ":" + objectName;
     }
+
+    /**
+     * Always returns null
+     */
+    @Override
+    public Integer getVersion(String objectSchema, String objectType, String objectName) {
+        return null;
```

Comment:
Is this really testing VersionHistory service? shouldn't it return -1 or something to confirm?

Reply:
We use the real VersionHistoryService for anywhere that needs a true version (like in the new DerbyMigrationTest).
Perhaps we could return a -1 (or a 0) here to avoid having the null checks on the `apply` methods, but I went with null since we were already using Integer arguments (and not ints).